service/history/configs/quotas.go

```diff
@@ -100,6 +100,7 @@ var (
 		"/temporal.server.api.historyservice.v1.HistoryService/ListQueues":                             1,
 		"/temporal.server.api.historyservice.v1.HistoryService/ListTasks":                              1,
 		"/temporal.server.api.historyservice.v1.HistoryService/CompleteNexusOperation":                 1,
+		"/temporal.server.api.historyservice.v1.HistoryService/HealthCheck":                            1,
```

Comment:
I guess you want HealthCheck to be high priority or be excluded from rate limiting.